tests/unit/insights/test_importer.py

```diff
@@ -629,6 +640,53 @@ def generate_insights(
         batch_insert_mock.assert_called_once()
         product_insight_delete_mock.assert_called_once()
 
+    def test_import_msc_insights(self, mocker):
+        class FakeImporter(InsightImporter):
```

Comment:
I don't think you need a FakeImporter for your test.
You are not testing the mechanics of the Genral Importer, you are testing the PackagerCodeImporter.